pcb/elec/src/rpi-hat.ato

```diff
@@ -0,0 +1,114 @@
+"""
+Raspberry Pi HAT PCB for sethfischer/rover.
+"""
+
+from "generics/capacitors.ato" import Capacitor
+from "generics/interfaces.ato" import I2C, Power
+from "generics/leds.ato" import LEDIndicatorGreen, LEDIndicatorRed
+from "generics/resistors.ato" import I2CPullup, Resistor
+
+from "elec/src/components/eeproms.ato" import CAT24C32WI_GT3
+from "elec/src/components/headers_rpi.ato" import HeaderGPIORaspberryPiTH
+from "elec/src/components/headers.ato" import Header2PinDipStraightTH
+
+
+module RpiHatIdEeprom:
+    """
+    Raspberry Pi HAT EEPROM
+
+    # Reference
+
+    * [Raspberry Pi HAT+ Specification](https://datasheets.raspberrypi.com/hat/hat-plus-specification.pdf)
+    * [Raspberry Pi HAT EEPROM circuit diagram](https://github.com/raspberrypi/hats/blob/5f2058bf8eebf43dd19d7218f1e38d14a9835231/eeprom-circuit.png)
+    """
+    # power
+    power_3v3 = new Power
+    signal gnd ~ power_3v3.gnd
+
+    # EEPROM I2C
+    signal i2c
+    i2c = new I2C
```

Comment:
Seems like both a `signal i2c` and an i2c interface are created here. I'd remove the signal if it is not being used. Made an issue for it: https://github.com/atopile/atopile/issues/222